lib/index.js

```diff
@@ -1,3 +1,3 @@
 const createSort = require('./create-sort');
 const loadConfig = require('./load-config');
-module.exports = createSort(loadConfig());
```

Comment:
And this too

Reply:
I find it less intuitive and more of a breaking change that way, but since you didn't respond i rollback the change as you asked.